---
layout: layouts/base.njk
eleventyNavigation:
  key: CV
  order: 2
---
# CV

----

## Education and work

- *2025 - Current* : **Postdoc**, University of Antwerp
- *2020 - 2025* : **PhD in Bioscience Engineering**, Ghent University
    - Dissertation title: Self-supervised Transformers for Biological Data Modalities
- *2018 - 2020* : **MSc in Bioinformatics (Bioscience Engineering)**, Ghent University
- *2015 - 2018* : **BSc in Bioscience Engineering**: Cell and Gene Biotechnology, Ghent University
- *2009 - 2015* : **Science-Mathematics**, Leiepoort Campus Sint-Hendrik Deinze

Additional courses:
- *2023*: ACDL (Advanced Course on Data Science & Machine Learning), 5-day course, Castiglione della Pescaia, Italy
- *2023*: Causal Machine Learning, semester course at Ghent University
- *2022*: Research to Market, 2-day course at Ghent University
----

## Academic output
- **Publications**:
    - **De Waele, Gaetan**, Jim Clauwaert, Gerben Menschaert, and Willem Waegeman. ["CpG Transformer for imputation of single-cell methylomes."](https://doi.org/10.1093/bioinformatics/btab746) Bioinformatics 38, no. 3 (2022): 597-603.
    - **De Waele, Gaetan**, Gerben Menschaert, and Willem Waegeman.["An antimicrobial drug recommender system using MALDI-TOF MS and dual-branch neural networks"](https://doi.org/10.7554/eLife.93242.4) eLife 13:RP93242 (2024).
    - Mey, Friederike*, **Gaetan De Waele***, Wouter Demeester, Chiara Guidi, Dries Duchi, Tomek Diederen, Hanne Kochuyt et al. ["Machine learning reveals novel compound for the improved production of chitooligosaccharides in Escherichia coli."](https://doi.org/10.1016/j.nbt.2025.01.005) New Biotechnology (2025).
    - **De Waele, Gaetan**, Gerben Menschaert, Peter Vandamme, and Willem Waegeman. ["Pre-trained Maldi Transformers improve MALDI-TOF MS-based prediction."](https://doi.org/10.1016/j.compbiomed.2025.109695) Computers in Biology and Medicine (2025).
    - Vergauwe, Fauve, **Gaetan De Waele**, Andrea Sass, Callum Highmore, Niall Hanrahan, Yoshiki Cook, Mads Lichtenberg et al. ["Harnessing machine learning to predict antibiotic susceptibility in Pseudomonas aeruginosa biofilms."](https://doi.org/10.1038/s41522-025-00833-4) npj Biofilms and Microbiomes 11, no. 1 (2025): 205.
<p>

- **Preprints**:
    - **De Waele, Gaetan**, Gerben Menschaert, and Willem Waegeman. ["A systematic assessment of single-cell language model configurations."](https://www.biorxiv.org/content/10.1101/2025.04.02.646825v1) bioRxiv (2025): 2025-04.

<p>

(*: shared first author)

- **Conference contributions**:
    - **Oral** at ISMB22, Madison, USA. "CpG Transformer for imputation of single-cell methylomes"
    - **Oral** at AAAI24 LLMs4Bio workshop, Vancouver, Canada. "Pre-trained Maldi Transformers improve MALDI-TOF MS-based prediction"
    - **Poster** at ABLS24, Leuven, Belgium. "Transformers for MALDI-TOF MS-based antimicrobial drug recommendation"
    - **Oral** at ISMB24, Montreal, Canada. "Transformers for MALDI-TOF MS-based antimicrobial drug recommendation"
    - **Poster** at VIB.AI Symposium 24, Mechelen, Belgium, "Benchmarking single-cell Language Model building blocks"

<p>

- **Software**:
    - [cpg-transformer](https://github.com/gdewael/cpg-transformer), code supporting my work on methylation (CpG) data imputation.
    - [h5torch](https://github.com/gdewael/h5torch), a simple utility that allows PyTorch Dataloading from HDF5 files.
    - [bio-attention](https://github.com/gdewael/bio-attention), definitions for transformers adapted to the work in my PhD.
    - [maldi-nn](https://github.com/gdewael/maldi-nn), code supporting my work on MALDI-TOF mass spectrometry.
    - [bento-sc](https://github.com/gdewael/bento-sc), code supporting my work on single-cell transcriptomic language models.
    - [cut2min-bucket](https://github.com/gdewael/cut2min-bucket), a small dataloading utility to use flash-attn with variable length inputs.



----

## Teaching

- *2020 - 2025* : **Teaching assistance** for "Machine learning for the Life Sciences" course
- *2024 - 2025* : **Tutor for Master thesis** "End-to-end de novo metabolomics" by Wout Welvaert
- *2024 - 2025* : **Tutor for Master thesis** "Variational inference for DTI" by Robbe Claeys
- *2023 - 2024* : **Tutor for Master thesis** "Novel neural networks for bacterial species prediction from MALDI-TOF data" by Jorge Isaac Cueva Villavicencio
- *2022 - 2023* : **Tutor for Master thesis** "Two-Branch Neural Networks for Predicting Protein-DNA Interaction" by Natan Tourne
- *2022 - 2023* : **Tutor for Master thesis** "Designing novel protein sequences using advanced generative models" by Scout Van den Bergh
- *2021 - 2022* : **Tutor for Master thesis** "Detection of 5mC modification in Nanopore sequencing data using deep learning" by Yari Van Laere
- *2021 - 2022* : **Teaching assistance** for "UGain Machine Learning" course
- *2021 - 2022* : **Tutor for MSc Bioinformatics Design Project** "Automated benchmarking of optimized general
purpose machine learning classifiers for single cell
transcriptomics" by Ruben Allaert, Seoyeon Oh, Natan Tourne, and Christopher Van Hessche
- *2020 - 2021* : **Tutor for MSc Bioinformatics Design Project** "Benchmarking study of ML methods for bacterial
species identification from MALDI-TOF MS data" by Kobe De Temmerman, Triana Forment, Ju Hyung Lee, Natalie Thomas, and Elien Martens




